index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles.css" rel="stylesheet">
+    <title>Portfolio: Mobile Version Skeleton</title>
+</head>
+<body>
+    <section class="mobile-skeleton">
+        <header class="header">
+            <nav class="navbar">
+                <ul class="header-container">
+                    <li class="logo"><h4>My Logo</h4></li>
+                    <li><img src="images/menu.png" alt="menu"></li>
+                </ul>
+            </nav>
+        </header>
+        <section class="headline">
+            <img src="images/headline.png" alt="headline">
+            <div class="main">
+                <p class="intro">I'm Kadenyi Glad to see you!</p>
+                <p class="pitch">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="connect">
```

Comment:
- [ ] please try to follow the figma characteristics of exact margins, font weight, line height, sizes, family, etc so it looks similar.
- your design 👇🏾 
![Screenshot (48)_LI](https://user-images.githubusercontent.com/65298048/170439066-fe779fe3-dc9b-4415-8570-2224b21f8489.jpg)
- figma design 👇🏾 
![Screenshot (39)_LI](https://user-images.githubusercontent.com/65298048/170439178-746077a4-f4f0-4523-8e96-0e3d6b47e699.jpg)